olm-catalog/serverless-operator/project.yaml

```diff
@@ -28,9 +28,9 @@ requirements:
     golang: '1.19'
     nodejs: 16.x
     ocpVersion:
-        min: '4.10'
+        min: '4.11'
         max: '4.13'
-        label: 'v4.10'
+        label: 'v4.11'
```

Comment:
Perhaps, also `requirements.kube.minVersion` should be increased to 1.24.0?

Reply:
👍